src/lexer.js

```diff
@@ -29,7 +33,16 @@ export function parse (source, name = '@') {
   return { exports: [...exports], reexports: [...reexports] };
 }
 
-function copy (src, outBuf16) {
+function copyBE (src, outBuf16) {
+  const len = src.length;
+  let i = 0;
+  while (i < len) {
+    const ch = src.charCodeAt(i);
+    outBuf16[i++] = (ch & 0xff) << 8 | (ch & 0xff0) >> 8;
```

Comment:
This is missing a `0`:
```suggestion
    outBuf16[i++] = (ch & 0xff) << 8 | (ch & 0xff00) >> 8;
```

Reply:
Thanks for spotting this, it wasn't causing an issue because the right shift drops thesebits anyway, the version on master has the fix.